xlsx/src/export/mod.rs

```diff
@@ -63,10 +64,14 @@ pub fn save_to_xlsx(model: &Model, file_name: &str) -> Result<(), XlsxError> {
     let writer = BufWriter::new(file);
     save_xlsx_to_writer(model, writer)?;
 
+    if let Err(err) = model.garbage_collector() {
```

Comment:
Running the garbage collector clean up after saving of a file to avoid an issue where you can't save due to GC failure.